src/components/TodoForm/TodoForm.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TodoForm.css';
+
+import { SelectOption } from '../SelectOption';
+
+export class TodoForm extends React.PureComponent {
+  state = {
+    todoTitle: '',
+    username: '',
+    todoUserId: 0,
+    todoId: this.props.todoId,
```

Comment:
it's a very bad practice to use props in state
https://medium.com/@justintulk/react-anti-patterns-props-in-initial-state-28687846cc2e

Reply:
just use todos.length + 1 for id, when you create a new todo, no need to put todoId in state